backend/src/db/providers/userProvider.ts

```diff
@@ -1,9 +1,12 @@
 import { ObjectId } from 'mongoose';
 
-import { IUser } from 'interfaces/Ientities/Iusers';
+import { IUserSkill } from 'interfaces/Ientities/IUserSkill';
 import BadRequestError from 'classes/errors/clientErrors/BadRequestError';
 import NotFoundError from 'classes/errors/clientErrors/NotFoundError';
 import UserModel from 'db/models/User';
+import UserSkillModel from 'db/models/UserSkill';
+import SkillGroupModel from 'db/models/SkillGroup';
+import SkillModel from 'db/models/Skill';
 
 const getUserProvider = async (userId: string) => {
```

Comment:
as we have getFullUserInfo do we need this provider ?

Reply:
Well, actually yes. The thing is that we use this one when trying to log in. As guys told me, they don't use skills data that we're sending to them during an attempt to log in. So there is no need to send unnecessary populated skills in my opinion.